recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -36,14 +37,16 @@ SRC_URI = "repo://github.com/xen-troops/manifests;protocol=https;branch=master;m
 ###############################################################################
 # these will be populated into the inner build system on do_unpack_xt_extras
 # N.B. xt_shared_env.inc MUST be listed AFTER meta-xt-prod-extra
```

Comment:
This comment line need to be rewritten, taking into account removal of xt_shared_env.inc.

Reply:
agree